src/components/header/examples/header-responsive.tsx

```diff
@@ -0,0 +1,92 @@
+import { Component, h } from '@stencil/core';
+import { Action } from '@limetech/lime-elements';
+/**
+ * How Responsive layout of header works
+ * However, sometimes you may need to make the layout be responsive and split
+ * into two rows, at a break point.
+ *
+ * To activate this responsive layout, you can simply add the `has-responsive-layout`
+ * class to your `limel-header` component.
+ *
+ * This makes a few changes in the layout. Firstly, both the left side (icon and
+ * headings) and right side (actions slot) will occupy 50% of the total header
+ * width each. However, the width of left and right side will never become smaller
+ * than `22rem`.
```

Comment:
I think the default breakpoint is 12rem

Reply:
No, the default is `352px` or `22rem` 👇

````scss
grid-template-columns: repeat(
        auto-fit,
        minmax(
            clamp(50%, var(--header-responsive-breakpoint, pxToRem(352)), 100%),
            1fr
        )
    );
````

However in the example I have overwritten in with `12rem`